Server/mods/deathmatch/logic/CMainConfig.cpp

```diff
@@ -268,7 +283,7 @@ bool CMainConfig::Load()
             if (!bFound)
                 CLogger::ErrorPrintf("Unknown client_file '%s'\n", *strName);
         }
-    } while (pNode);
+    } while (currentNode);
```

Comment:
you can do the same with this loop as in the case above

Reply:
Check now